nb_conda/handlers.py

```diff
@@ -259,7 +259,7 @@ def get(self):
 
 # there is almost no text that is invalid, but no hyphens up front, please
 # neither all these suspicious but valid caracthers...
-_env_regex = r"(?P<env>[^/&+$?@<>%*-][^/&+$?@<>%*]*)$"
+_env_regex = r"(?P<env>[^/&+$?@<>%*-][^/&+$?@<>%*]*)"
```

Comment:
I tried to create an environment with the name `myenv%two` and it failed to create it, which I think is appropriate... and the same with `%myenv` so I think is working properly...